.travis.yml

```diff
@@ -1,4 +1,16 @@
 sudo: false
 
-language: node_js
-node_js: stable
+language: java
```

Comment:
Unfortunately this did not seem to work. Investigation led to this travis issue:
https://github.com/travis-ci/travis-ci/issues/3827

An alternative could be using node_js language and then manually hooking up java.